makefiles/tools/openocd-adapters/mulle.inc.mk

```diff
@@ -0,0 +1,33 @@
+# OpenOCD settings for Mulle programmer board.
+# Try to determine which OpenOCD config file we should use based on the
+# programmer board serial number.
+
+# Fall back to PROGRAMMER_SERIAL for backwards compatibility
+export DEBUG_ADAPTER_ID ?= $(PROGRAMMER_SERIAL)
```

Comment:
I don't know if for this just an announce that it changed would be enough... but let's keep it for now...